app/ui/persmission-handler.d.ts

```diff
@@ -0,0 +1,36 @@
+export class PermissionHandler {
```

Comment:
`app/ui` shouldn't exist. All type definitions created or generated should be contained within a `types` folder in your react lib (`remix-project/libs/remix-ui/plugin-manager/src/lib/types`)

Reply:
Done.